src/language/rule/rule.ts

```diff
@@ -79,10 +79,10 @@ export class RuleFailure {
     private ruleName: string;
 
     constructor(sourceFile: ts.SourceFile,
-                start: number,
-                end: number,
-                failure: string,
-                ruleName: string) {
+        start: number,
```

Comment:
Eh, I don't like this one. It goes against the align rule and I like the readability we had here.